src/SFA.DAS.Roatp.CourseManagement.Web.UnitTests/Services/SessionServiceTests/SessionServiceSetTests.cs

```diff
@@ -27,97 +27,110 @@ public void Before_Each_Test()
         }
 
         [Test, AutoData]
-        public void Set_AddsValueToSession(string key, string context, string value)
+        public void Set_AddsValueToSession(string key, string value)
```

Comment:
Can you please add a test for each Set and Set<T> to prove that is overrides any existing value